src/Database/LibSQLConnection.php

```diff
@@ -68,6 +42,13 @@ public function select($query, $bindings = [], $useReadPdo = true)
         return $resultArray;
     }
 
+    protected function getDefaultQueryGrammar()
```

Comment:
This just fixes grammar, can be found in https://github.com/tursodatabase/turso-driver-laravel/pull/12